src/Appwrite/Utopia/Response/Model/HealthStatus.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class HealthStatus extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('checkTime', [
```

Comment:
```suggestion
            ->addRule('ping', [
```

why don't we just call it ping ? which is a much more familiar term to most developers.

Reply:
We can.